contact.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.css"
+    />
+    <title>BariCare - Contact</title>
+  </head>
+    <body>
+        <div class="wrapper">
+        <!-- Header (Logo + Nav Menu)-->
+        <header>
+            <span class="logo">
+            <img
+                src="/images/baricare_logo.svg"
+                class="logo-svg"
+                alt="BariCare Logo"
+            />
+            <p>Care</p>
+            </span>
+            <input type="checkbox" class="nav-toggle" id="nav-toggle" />
+            <label
+            for="nav-toggle"
+            class="nav-toggle-label"
+            aria-label="Menu button"
+            ></label>
+            <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="gastric_sleeve.html">Gastric sleeve</a></li>
+                <li><a href="gastric_bypass.html">Gastric bypass</a></li>
+                <li><a href="gastric_band.html">Gastric band</a></li>
+                <li><a href="contact.html" class="active" >Contact</a></li>
+            </ul>
+            </nav>
+        </header>
+
+        <main class="main-content contact">
+            <!-- Breadcrumb navigation-->
+            <ul class="breadcrumb">
+              <li><a href="index.html" class="breadcrumb-item">Home</a></li>
+              <li>
+                <a href="#" class="breadcrumb-item current-page"
+                  ><span aria-current="page">Contact</span></a
+                >
```

Comment:
Formatează codul mai frumos

Reply:
O formatare de genul acesta aș vrea:

```xml
<Bunicu>
  <!-- Toți copii elementului au două spații în plus -->
  <Unchiu />
  <Tata>
    <!-- Toți copii elementului au două spații în plus -->
    <AlaMicu />
    <Frate>
      <!-- Și textul se consideră „copil” -->
      Niște text
    </Frate>
  <Tata>
</Bunicu>
```

Reply:
Cam asa?

`<nav>
            <ul>
              <li>
                <a href="index.html">Home</a>
              </li>
              <li>
                <a href="gastric_sleeve.html">Gastric sleeve</a>
              </li>
              <li>
                <a href="gastric_bypass.html">Gastric bypass</a>
              </li>
              <li>
                <a href="gastric_band.html">Gastric band</a>
              </li>
              <li>
                <a href="contact.html" class="active" >Contact</a>
              </li>
            </ul>
        </nav>`

Reply:
Da. Văd că sunt mici probleme și în celelalte fișiere.

Reply:
Intr-adevar, am incercat sa reformatez codul in toate paginile.